server/middleware/checkUserId.js

```diff
@@ -0,0 +1,14 @@
+import validation from './validation';
+import messages from '../utils/messages';
+
+module.exports = (req, res, next) => {
```

Comment:
use `export` and `export default` instead of `module.exports`

Reply:
Noted.